src/assets/scripts/client/navigationLibrary/Route/RouteModel.js

```diff
@@ -135,15 +135,28 @@ export default class RouteModel extends BaseModel {
     }
 
     /**
-     * Verify that a routeCode has exactly 3 segments
+     * Verify that a routeString has exactly 3 segments
      *
      * @for RouteModel
      * @method _isValidRouteCode
-     * @param routeCode {string}
+     * @param routeString {string}
      * @return {boolean}
      * @private
      */
-    _isValidRouteCode(routeCode) {
-        return routeCode.split(SEGMENT_SEPARATION_SYMBOL).length === MAXIMUM_ROUTE_SEGMENT_LENGTH;
+    _isValidRouteCode(routeString) {
+        return RouteModel.isProcedureRouteString(routeString);
     }
```

Comment:
TBH, we aren't (at the moment).  But we will in the next sprint.  This RouteModel thing may have reached the end of its life, as it is now.  This class is dealing with, specifically, _procedure_ routeStrings.  thats why there is a static method, so we can check those things before they even get into the class.

These types of things are going to be cleaned significantly in the next sprint with the introduction of the `RouteBuilder` and such.  The `RouteModel` will likely change shape considerably.  Not that I'm typing, this may actually become something like `ProcedureRouteModel` or something.  I'm not sure yet.